pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -575,5 +578,31 @@ private void clearUnAckedMsgs(Consumer consumer) {
         subscription.addUnAckedMessages(-unaAckedMsgs);
     }
 
+    public static class SendMessageInfo {
```

Comment:
At dispatcher, we want to know how many messages/bytes we actually sent to manage the permits. Therefore, at here in consumer, we exactly know how many msgs/bytes is sent and return it to dispatcher. Earlier, `consumer.sendMessages()` was returning `Pair<ChannelPromise, Integer>` and we wanted to add one more variable so, created `SendMessageInfo` entity to combine all information.